.env

```diff
@@ -1 +1 @@
-MONGOHQ_URL = localhost
\ No newline at end of file
+MONGOHQ_URL = localhost
```

Comment:
we do not need to commit .env file, just add a .env.sample that contains dummy values for the env vars

Reply:
@nedemenang  has been saddled with the responsibility of removing the .env and to create the .env.sample file, i that will reflect when his PR is marged